packages/@ember/-internals/glimmer/lib/modifiers/on.ts

```diff
@@ -268,7 +268,7 @@ function addEventListener(
   For example, in our example case above if you'd like to pass in the post that
   was being liked when the button is clicked you could do something like:
 
-  ```app/templates/components/like-post.js
+  ```app/templates/components/like-post.hbs
```

Comment:
Small suggestion since documentation should use co-location from now on:

```suggestion
  ```app/components/like-post.hbs
```